### DISCUSSION TITLE
[IntelliSense] Extension doesn't work when the tailwind.config file is .ts instead of .js

### DISCUSSION THREAD

**Discussion by matthiasfeist** [tailwindlabs/tailwindcss#11789] [Help]
**What version of VS Code are you using?**
Version: 1.81.0 (Universal)
Commit: 6445d93c81ebe42c4cbd7a60712e0b17d9463e97
Date: 2023-08-02T12:40:02.782Z
Electron: 22.3.18
ElectronBuildId: 22689846
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.5.0

**What version of Tailwind CSS IntelliSense are you using?**
v0.9.11

**What version of Tailwind CSS are you using?**
`"tailwindcss": "3.3.3"`

**What package manager are you using?**
npm

**What operating system are you using?**
macOS

**Tailwind config**
Unchanged from here: https://github.com/vercel/next.js/blob/canary/packages/create-next-app/templates/default-tw/ts/tailwind.config.ts

**VS Code settings**
unchanged default settings

**Reproduction URL**
https://github.com/vercel/next.js/blob/canary/packages/create-next-app/templates/default-tw/ts/tailwind.config.ts

**Describe your issue**
When creating a new NextJS project with the CLI wizard and then choosing TypeScript and TailwindCSS in the process, the created tailwind.config file as a .ts extension instead of a .js extension. Therefore the VSCode extension doesn't work

**Comment by bradlc** [comment]
Hey @matthiasfeist. I have converted this to a discussion until we can identify an issue with the extension itself.

Tailwind CSS IntelliSense does support `.ts` config files and we have used these ourselves in many Next.js projects without issue.

Please can you share the logs from the extension's output panel? (you can reveal this using the `Tailwind CSS: Show Output` command)

**Comment by matthiasfeist** [comment]
Now it works for me. Not sure what changed TBH. Sorry for the fuzz :/
